sv-disable

```diff
@@ -1,20 +1,28 @@
-#!/bin/sh
+#!/usr/bin/env sh
 
-show_help(){
+default_vars() {
+    local prefix="${TERMUX__PREFIX:-"${PREFIX}"}"
 
-    printf 'Usage:
+    : "${LOGDIR:="${prefix}/var/log"}"
+    export LOGDIR
+
+    : "${SVDIR:="${prefix}/var/service"}"
+    export SVDIR
+}
+
+show_help() {
+    printf -- 'Usage:
     sv-disable <service-name>
 
     Disabling a service will make it not autostart
 '
 }
+
 if [ -z "$1" ];then
     show_help
     exit 1
 fi
-if [ -u "$SVDIR" ];then
-    >&2 echo "Error: SVDIR not set"
-    exit 1
-fi
-touch "$SVDIR/$1/down"
-sv down $1
+
+default_vars
+: >> "${SVDIR}/${1}/down"
```

Comment:
Why not use `touch` here to create an empty file?
`: >>` is idiomatically equivalent to `touch`.
But harder to read at a glance.